src/FixedDataTable.js

```diff
@@ -423,9 +419,16 @@ var FixedDataTable = React.createClass({
   },
 
   componentWillReceiveProps(/*object*/ nextProps) {
+
+    // If scrollLeft is unchanged, ignore it
```

Comment:
this is a bit weird, but seemed best since I don't have access to previous props in the redux store and don't want to rely on it.